python/ray/tune/tune.py

```diff
@@ -555,6 +556,7 @@ class and registered trainables.
         )
 
     ray._private.usage.usage_lib.record_library_usage("tune")
+    air_usage.tag_env_vars()
```

Comment:
Might be good to add a comment here or in the PR description explaining why this is placed here vs. anywhere else in the stack.